tox.ini

```diff
@@ -20,17 +20,18 @@ allowlist_externals=make,pre-commit
 install_command=python -m pip install {opts} {packages}
 usedevelop=True
 commands=
+    # Note: `pytest-xdist` n=10 seems to cause connection issues with geth; keep it at 6 for now
     core: pytest {posargs:tests/core -m "not asyncio"}
     core_async: pytest {posargs:tests/core -m asyncio}
-    ens: pytest {posargs:tests/ens --ignore=tests/ens/normalization/test_normalize_name_ensip15.py}
-    ensip15: pytest {posargs:tests/ens/normalization/test_normalize_name_ensip15.py -q}
-    integration-goethereum-ipc: pytest {posargs:tests/integration/go_ethereum/test_goethereum_ipc.py -k "not Async"}
-    integration-goethereum-ipc_async: pytest {posargs:tests/integration/go_ethereum/test_goethereum_ipc.py -k Async}
-    integration-goethereum-http: pytest {posargs:tests/integration/go_ethereum/test_goethereum_http.py -k "not Async"}
-    integration-goethereum-http_async: pytest {posargs:tests/integration/go_ethereum/test_goethereum_http.py -k Async}
-    integration-goethereum-legacy_ws: pytest {posargs:tests/integration/go_ethereum/test_goethereum_legacy_ws.py}
-    integration-goethereum-ws: pytest {posargs:tests/integration/go_ethereum/test_goethereum_ws}
-    integration-ethtester: pytest {posargs:tests/integration/test_ethereum_tester.py}
+    ens: pytest {posargs:tests/ens --ignore=tests/ens/normalization/test_normalize_name_ensip15.py -n 10}
+    ensip15: pytest {posargs:tests/ens/normalization/test_normalize_name_ensip15.py -q -n 10}
+    integration-goethereum-ipc: pytest {posargs:tests/integration/go_ethereum/test_goethereum_ipc.py -k "not Async" -n 10}
+    integration-goethereum-ipc_async: pytest {posargs:tests/integration/go_ethereum/test_goethereum_ipc.py -k Async -n 10}
+    integration-goethereum-http: pytest {posargs:tests/integration/go_ethereum/test_goethereum_http.py -k "not Async" -n 10}
+    integration-goethereum-http_async: pytest {posargs:tests/integration/go_ethereum/test_goethereum_http.py -k Async -n 10}
+    integration-goethereum-legacy_ws: pytest {posargs:tests/integration/go_ethereum/test_goethereum_legacy_ws.py -n 10}
+    integration-goethereum-ws: pytest {posargs:tests/integration/go_ethereum/test_goethereum_ws -n 10}
+    integration-ethtester: pytest {posargs:tests/integration/test_ethereum_tester.py -n 10}
```

Comment:
I wonder if auto would be better here? then we're not hard coding a certain number.

Reply:
My only hesitation was that `auto` could end up being _more_ than `10`. I was sort of trying to set some lower limit. But I guess if it can handle more than `10` that would be ok?

Reply:
Thoughts on that?

Reply:
I added it, let's see what auto does